.github/workflows/release_test_sfdx.yml

```diff
@@ -47,6 +47,12 @@ jobs:
                   python-version: 3.11
                   cache: pip
                   cache-dependency-path: "pyproject.toml"
+            - name: Check pip cache existence
+              run: |
+                  if [ ! -d ~/.cache/pip ]; then
+                      echo "No pip cache directory found, skipping cache step."
+                  fi
+
```

Comment:
I think we can remove it entirely:
```suggestion
```
See: https://docs.astral.sh/uv/guides/integration/github/#caching

(We need to do a bit of CI cleanup.)